core/deployment/src/test/java/io/quarkus/deployment/proxy/SimpleClassProxyTest.java

```diff
@@ -27,7 +29,7 @@ public void testProxyCreation() throws InstantiationException, IllegalAccessExce
         assertMethod3(instance);
         assertMethod4(instance);
 
-        assertThat(invocationHandler.invocationCount).isEqualTo(4);
+        assertThat(invocationHandler.invocationCount).isEqualTo(EXPECTED_INVOCATION_COUNT);
```

Comment:
Thanks for the effort, but I don't see it as an improvement: when reading the tests, you actually want to see the value directly, not check a constant.

It's also more annoying when you adjust the test.